### DISCUSSION TITLE
Added Enemies faction ?

### DISCUSSION THREAD

**Discussion by KydasadyK** [Vdauphin/HeartsAndMinds#1289] [Question and answer]
Hello !
I would like to know if it's possible to add enemies because I changed my mod pack and I removed the cup and CFP to have more space but I don't have the new enemies from other mods... I put opfort project which weighs less but not enough choice in choice lines and Iraqi-Syrian Conflict

**Comment by Vdauphin** [comment]
Hello,

The mission is persistant so once a city get initialize (the first time you enter it), all units inside it will be saved
The only proper way is to delete the database

But there is a workaround, you can reset saved unit inside each city by doing something like that in the debug console (exec server):
```sqf
{
    _x setVariable ["data_units", []];
} forEach btc_city_all;
```
Cheers

**Reply by KydasadyK** [reply]
It's not in the root of your files or you have to add them? because I'm an editor and I prepare it for an open server

**Reply by Vdauphin** [reply]
> because I'm an editor and I prepare it for an open server

Well, if you didn't start your campaign you don't need those lines. I thought you were changing mod during your campaign

I don't get your problem...
